lib/mpl_toolkits/axes_grid1/inset_locator.py

```diff
@@ -504,9 +503,10 @@ def inset_axes(parent_axes, width, height, loc='upper right',
     if bbox_transform in [parent_axes.transAxes,
                           parent_axes.figure.transFigure]:
         if bbox_to_anchor is None:
-            warnings.warn("Using the axes or figure transform requires a "
-                          "bounding box in the respective coordinates. "
-                          "Using bbox_to_anchor=(0,0,1,1) now.")
+            cbook._warn_external("Using the axes or figure transform "
+                                 "requires a bounding box in the respective "
+                                 "coordinates. Using bbox_to_anchor=(0,0,1,"
```

Comment:
I wouldn't break the parenthesis for better readability in the code. Would break befrore "Using" here even though the line will get quite short.